django/contrib/postgres/fields/array.py

```diff
@@ -70,6 +70,25 @@ def check(self, **kwargs):
                 )
         return errors
 
+    def _check_default(self):
+        if self.has_default() and self.default is not None and not callable(self.default):
+            return [
+                checks.Warning(
+                    "ArrayField uses a default instance that's shared between all field instances.",
+                    hint="Use a callable instead, e.g., instead of `[]`, use `list`",
```

Comment:
I'd reverse the ordering and say "use list instead of []"

Reply:
Will fix this now